logbook-servlet/src/test/java/org/zalando/logbook/servlet/FormRequestTest.java

```diff
@@ -0,0 +1,118 @@
+package org.zalando.logbook.servlet;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.RequestPostProcessor;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.zalando.logbook.DefaultHttpLogFormatter;
+import org.zalando.logbook.DefaultSink;
+import org.zalando.logbook.HttpLogFormatter;
+import org.zalando.logbook.HttpLogWriter;
+import org.zalando.logbook.Logbook;
+import org.zalando.logbook.Precorrelation;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.startsWith;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.zalando.logbook.servlet.FormRequestMode.BODY;
+import static org.zalando.logbook.servlet.FormRequestMode.OFF;
+import static org.zalando.logbook.servlet.FormRequestMode.PARAMETER;
+
+/**
+ * Verifies that {@link LogbookFilter} delegates to {@link HttpLogWriter} correctly.
+ */
+final class FormRequestTest {
+
+    private final HttpLogFormatter formatter = spy(new ForwardingHttpLogFormatter(new DefaultHttpLogFormatter()));
+    private final HttpLogWriter writer = mock(HttpLogWriter.class);
+
+    @BeforeEach
+    void setUp() {
+        reset(formatter, writer);
```

Comment:
Why is such `reset` required?

Reply:
Nope, not needed. Probably a left over from an even earlier version of this test. Or just copy&pasted from somewhere else without giving it another though.

Reply:
Ok thanks, I had a short panic attack that with JUnit 5 the test class life cycle has changed :-D